python/ray/rllib/utils/actors.py

```diff
@@ -36,7 +36,11 @@ def completed_prefetch(self):
 
         for worker, obj_id in self.completed():
             plasma_id = ray.pyarrow.plasma.ObjectID(obj_id.id())
-            ray.worker.global_worker.plasma_client.fetch([plasma_id])
+            if not ray.global_state.use_raylet:
```

Comment:
Thanks for fixing this!

Reply:
@ericl Sure. Unfortunately, this won't fix the A3C crash. Maybe you and @pcmoritz have some clue. ;-)